Tests/conf.json

```
@@ -51,7 +51,7 @@
             "fromversion": "5.0.0"
         },
         {
-            "integrations": "Azure Active Directory Identity Protection",
+            "integrations": "Azure Active Directory Identity and Access",
```

The name of the integration wasn't changed.
```suggestion
            "integrations": "Azure Active Directory Identity Protection",
```